internal/controllers/synthesis/lifecycle.go

```diff
@@ -136,15 +136,17 @@ func (c *podLifecycleController) Reconcile(ctx context.Context, req ctrl.Request
 	}
 
 	// Bail if it isn't time to synthesize yet, or synthesis is already complete
-	if comp.Status.CurrentSynthesis == nil || comp.Status.CurrentSynthesis.UUID == "" || comp.Status.CurrentSynthesis.Synthesized != nil || comp.DeletionTimestamp != nil {
+	if comp.Status.InFlightSynthesis == nil || comp.Status.InFlightSynthesis.UUID == "" || comp.Status.InFlightSynthesis.Synthesized != nil || comp.DeletionTimestamp != nil {
 		return ctrl.Result{}, nil
 	}
+	logger = logger.WithValues("synthesisID", comp.Status.InFlightSynthesis.UUID)
+	ctx = logr.NewContext(ctx, logger)
 
 	// Back off to avoid constantly re-synthesizing impossible compositions (unlikely but possible)
 	if shouldBackOffPodCreation(comp) {
 		const base = time.Millisecond * 250
-		wait := base * time.Duration(comp.Status.CurrentSynthesis.Attempts)
-		nextAttempt := comp.Status.CurrentSynthesis.PodCreation.Time.Add(wait)
+		wait := base * time.Duration(comp.Status.InFlightSynthesis.Attempts)
```

Comment:
I think we had this conversation already, but could you remind me why we're not doing exponential here?

Reply:
No reason that I'm aware of lol

I have a half working PR that integrates the scheduling controller into the retry loop so retries will be prioritized correctly. That one will also switch to exponential backoff.